src/promptflow/promptflow/_sdk/_serving/flow_invoker.py

```diff
@@ -41,6 +46,12 @@ class FlowInvoker:
     :type streaming: Union[Callable[[], bool], bool], optional
     :param connections: Pre-resolved connections used when executing, defaults to None
     :type connections: dict, optional
+    :param connections_name_overrides: The connection name overrides, defaults to None
```

Comment:
please add an example for the name overrides.

Reply:
add an example